integration_tests/src/main/python/orc_write_test.py

```diff
@@ -26,10 +26,17 @@
 pytestmark = pytest.mark.nightly_resource_consuming_test
 
 orc_write_basic_gens = [byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
-        string_gen, boolean_gen, DateGen(start=date(1590, 1, 1)),
+        string_gen, BooleanGen(nullable=False), DateGen(start=date(1590, 1, 1)),
```

Comment:
This is removing the test for nullable boolean values. Can we have an explicit test(s) that have a non-nullable struct with nullable values, or many different types, in it? I am fine if this is a follow on issue.